README.md

```diff
@@ -12,4 +12,4 @@ To run PathWeaver use the command `./gradlew run`
 
 
 ### Requirements
-- [JDK 11](http://jdk.java.net/11/)
+- [JDK 11](https://www.oracle.com/java/technologies/javase-jdk11-downloads.html)
```

Comment:
What do you think about pointing to Adopt that way the download is not behind a login?